eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -135,12 +137,13 @@ private void deleteCacheFiles() {
         sharedPreferences.edit().clear().commit();
     }
 
-    private void initClient(String host, boolean throwOnCallackError, boolean shouldDeleteCacheFiles, boolean isGracefulMode)
-            throws InterruptedException {
+    private void initClient(String host, boolean throwOnCallackError, boolean shouldDeleteCacheFiles, boolean isGracefulMode) {
         if (shouldDeleteCacheFiles) {
             deleteCacheFiles();
         }
 
+        CountDownLatch lock = new CountDownLatch(1);
```

Comment:
Moving this here I _think_ will deflake the tests 🤞 

My suspicion is that it was being instantiated once for _all tests_ so by the time the cache test came around, we didn't actually wait for the cache to hydrate flags before proceeding to request assignments.